core/utils.py

```diff
@@ -623,6 +623,20 @@ def get_time_in_millisecs(datetime_obj: datetime.datetime) -> float:
     return msecs + (datetime_obj.microsecond / 1000.0)
 
 
+def convert_millisecs_time_to_datetime_object(
+        date_time_msecs: float) -> datetime.datetime:
+    """Returns the datetime object from the given date time in milliseconds.
+
+    Args:
+        date_time_msecs: float. Date time represented in milliseconds.
+
+    Returns:
+        datetime. An object of type datetime.datetime corresponding to
+        the given milliseconds.
+    """
+    return datetime.datetime.fromtimestamp(date_time_msecs / 1000.0)
```

Comment:
How are timezones handled here -- does this give the datetime a timezone offset of 0 UTC?